app/crypto.go

```diff
@@ -0,0 +1,79 @@
+package app
+
+/* CHECKLIST
+ * [x] Uses interfaces as appropriate
+ * [x] Private package variables use underscore prefix
+ * [x] All parameters validated
+ * [x] All errors handled
+ * [x] Reviewed for concurrency safety
+ * [x] Code complete
+ * [x] Full test coverage
+ */
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/md5"
+)
+
+func EncryptWithAES256UsingPassphrase(bytes []byte, passphrase []byte) ([]byte, error) {
+	if len(bytes) == 0 {
```

Comment:
after discussion with @darinkrauss the bytes being short is not an issue and also I  understand this is a utility that the passphrase would be coming from config and if we wanted to restrict we would place the check elsewhere